pod-security/mutate/set-run-as-non-root/set-run-as-non-root.yaml

```diff
@@ -0,0 +1,87 @@
+apiVersion: kyverno.io/v1
+kind: ClusterPolicy
+metadata:
+  name: set-run-as-non-root
+  annotations:
+    policies.kyverno.io/title: Set runAsNonRoot
+    policies.kyverno.io/category: Pod Security Standards (Mutate)
+    policies.kyverno.io/severity: medium
+    policies.kyverno.io/subject: Pod
+    kyverno.io/kyverno-version: 1.6.0
+    kyverno.io/kubernetes-version: "1.22-1.30"
+    policies.kyverno.io/description: >-
+      Containers must be required to run as non-root users. This policy ensures
+      `runAsNonRoot` is set to `true`.
+spec:
+  validationFailureAction: Audit
+  background: true
+  rules:
+    - name: set-containers-run-as-non-root-to-true
+      match:
+        all:
+          - resources:
+              kinds:
+                - Pod
+      preconditions:
+        all:
+          - key: "{{ type(request.object.spec.containers) }}"
+            operator: Equals
+            value: "array"
```

Comment:
This is unnecessary. `containers[]` is a required object for a Pod and is always an array.

Reply:
Yes, in that case you would.